### PR TITLE
Hep center v2

### PR DESCRIPTION
**Pull request by sladan-hedvig**
- Added banner
- Send topic in the chat if we are coming from the help center

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

**Comment by julia-andersson** [comment]
Simulator Screenshot - iPhone 15 Pro - 2024-02-05 at 11 36 32

File: Projects/hCoreUI/Sources/MarkdownTextView.swift [hunk 1]
```diff
@@ -98,10 +74,10 @@ class CustomTextView: UIView, UITextViewDelegate {
 
     func setContent(from text: String) {
         configureTextView()
-        let schema = ColorScheme(UITraitCollection.current.userInterfaceStyle) ?? .light
+        let schema = ColorScheme.init(UITraitCollection.current.userInterfaceStyle) ?? .light
         let markdownParser = MarkdownParser(
-            font: Fonts.fontFor(style: fontStyle),
-            color: hTextColor.secondary.colorFor(schema, .base).color.uiColor()
+            font: Fonts.fontFor(style: config.fontStyle),
```
Comment (julia-andersson) [inline Projects/hCoreUI/Sources/MarkdownTextView.swift:79]: Should be 18px always, changed this before. Have discussed this with William.

File: Projects/hCoreUI/Sources/MarkdownTextView.swift [hunk 2]
```diff
@@ -115,17 +91,17 @@ class CustomTextView: UIView, UITextViewDelegate {
 
     private func calculateHeight() {
         let newHeight = getHeight()
-        self.snp.makeConstraints { make in
-            make.height.equalTo(newHeight)
-            make.width.equalTo(fixedWidth)
-        }
-        DispatchQueue.main.asyncAfter(deadline: .now() + 0.01) { [weak self] in
+        //        self.snp.makeConstraints { make in
```
Comment (julia-andersson) [inline Projects/hCoreUI/Sources/MarkdownTextView.swift:94]: remove

File: Projects/hCoreUI/Sources/MarkdownTextView.swift [hunk 3]
```diff
@@ -142,9 +118,37 @@ class CustomTextView: UIView, UITextViewDelegate {
...
+    let fontStyle: HFontTextStyle
+    let color: any hColor
+    let linkColor: any hColor
+    let linkUnderlineStyle: NSUnderlineStyle?
+    let onUrlClicked: (_ url: URL) -> Void
+
+    public init(
+        text: Markdown,
+        fixedWidth: CGFloat,
+        fontStyle: HFontTextStyle,
```
Comment (julia-andersson) [inline Projects/hCoreUI/Sources/MarkdownTextView.swift:140]: remove since we always want to use 18px

**Review by julia-andersson — CHANGES_REQUESTED** [review]

**Comment by sladan-hedvig** [comment]
> Not sure if this is correct maybe but we have different designs in figma and in the app on the deep link to help center in the banner
> 
> App: Simulator Screenshot - iPhone 15 Pro - 2024-02-05 at 11 36 32
> 
> Figma: Screenshot 2024-02-05 at 11 38 52

We discussed and changed to what we have now 👍

File: Projects/hCoreUI/Sources/MarkdownTextView.swift (hunk 1 above)
Comment (sladan-hedvig) [reply Projects/hCoreUI/Sources/MarkdownTextView.swift:79]: Lets keep it like this, on this way we can use it everywhere. I have placed everywhere else to 18, but for chat banner it is 14.

**Review by julia-andersson — APPROVED** [review]

**Pull request merged**